stable/prometheus-operator/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -70,4 +70,56 @@ rules:
   - get
   - list
   - watch
+---
```

Comment:
The recommendation is to have a single resource per file, or use a list to wrap them, rather than separating with `---`